test/conv2d_bias.cpp

```diff
@@ -35,7 +35,7 @@ struct conv2d_bias_driver : public conv_bias_driver<T>
                    tensor_elem_gen_checkboard_sign{}(is...);
         };
 
-        this->add(this->output, "output", this->get_tensor(get_inputs, gen_value));
+        this->add(this->output, "output", this->get_tensor(get_inputs<int>, gen_value));
```

Comment:
Does this change relate to the purpose of the given PR?

Reply:
Yep, it won't compile because it can't inference a type even if there is default one.